output-of-_settings-knn-index.txt

```diff
@@ -0,0 +1,32 @@
+curl -XPUT $ENDPOINT/test-index -H 'Content-Type: application/json' -d'
```

Comment:
I'll remove this file after I get some feedback. The point of the file is to show the settings difference after adding this change for knn and non-knn indices. The merge policy is updated here:
"merge":{"policy":{"floor_segment":"16mb"}}

Reply:
Sure, moved to attachments in description with a brief summary.